plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Core/Instance/Eq.hs

```diff
@@ -35,6 +36,10 @@ type HashableTermConstraints uni fun ann =
   , Hashable fun
   )
 
+-- TODO: dubious
```

Comment:
Not sure what to do about this. I could remove it, but then I would need to manually write an instance for `Term`, which would be okay but kind of annoying.

Reply:
Could use [vector-instances](https://hackage.haskell.org/package/vector-instances). Although given that it implements the instance the same way, I think simply keeping the instance is fine. Worth a comment though.